.pipelines/templates/PackageBuild.yml

```
@@ -201,6 +209,7 @@ steps:
         SPECS_DIR="${{ parameters.buildRepoRoot }}/${{ parameters.specsFolderPath }}" \
         SRPM_PACK_LIST="${{ parameters.srpmPackList }}" \
         $delta_fetch_arg \
+        $max_cascading_rebuilds_arg \
```

Since the toolkit default is an empty string as well, would this work:
```suggestion
        MAX_CASCADING_REBUILDS="${{ parameters.maxCascadingRebuilds }}" \
```

This way we could drop the `if` check. I'd then only move this next to the other arguments set this way.

---

Hmm... that should work.